cpp/tests/strings/booleans_tests.cpp

```diff
@@ -21,6 +21,8 @@
 #include <cudf_test/column_utilities.hpp>
 #include <cudf_test/column_wrapper.hpp>
 
+#include <thrust/iterator/transform_iterator.h>
+
```

Comment:
is this include needed? there are no other changes in this file.

Reply:
No I inserted manually.